provider/machine_pool_resource.go

```diff
@@ -96,17 +96,17 @@ func (t *MachinePoolResourceType) GetSchema(ctx context.Context) (result tfsdk.S
 				},
 			},
 			"autoscaling_enabled": {
-				Description: "Enables autoscaling.",
+				Description: "Enables autoscaling. This variable requires you to set a maximum and minimum replicas range using the `max_replicas` and `min_replicas` variables.",
 				Type:        types.BoolType,
 				Optional:    true,
 			},
 			"min_replicas": {
-				Description: "Min replicas.",
+				Description: "The minimum number of replicas for autoscaling.",
 				Type:        types.Int64Type,
 				Optional:    true,
 			},
 			"max_replicas": {
-				Description: "Max replicas.",
+				Description: "The maximum number of replicas not exceeded by the autoscaling functionality.",
```

Comment:
Again, we probably don't need the "not exceeded by" here. Unless it is part of code, it is implied by the use of "maximum".